low-level/files/opt/seagate/sspl/setup/manifest_support_bundle

```diff
@@ -23,39 +23,42 @@ import sys
 import socket
 import json
 import subprocess
+import warnings
+from framework.utils.conf_utils import (GLOBAL_CONF, Conf,
+    STORAGE_TYPE_KEY, SITE_ID_KEY, RACK_ID_KEY,
+    NODE_ID_KEY, CLUSTER_ID_KEY)
+from framework.base.sspl_constants import DATA_PATH
 
-from cortx.utils.conf_store import Conf
-from framework.utils.conf_utils import Utility
-from framework.base.sspl_constants import (PRODUCT_FAMILY,
-                                        RESOURCE_HEALTH_VIEW,
-                                        sspl_config_path,
-                                        GLOBAL_CONFIG_INDEX)
+warnings.filterwarnings(action='ignore', module='cryptography')
 
-SSPL_CONF = "SSPL_CONFIG"
-Conf.load(SSPL_CONF, sspl_config_path)
-global_config_url = Conf.get(SSPL_CONF, "SYSTEM_INFORMATION>global_config_copy_url")
-Conf.load(GLOBAL_CONFIG_INDEX, global_config_url)
+ManifestData = f"{DATA_PATH}manifest"
+if not os.path.exists(ManifestData):
```

Comment:
Create the "support_bundle" directory under DATA_PATH if already not exist and put all temporary files inside that.
once the script execution has done then clean up the temp files from a directory.
**Note**: Do not delete the "support_bundle" directory/folder because this folder will use the SSPL support bundle also. just delete the content of that folder that you had created.

Reply:
Done.
Inside the support_bundle dir, manifest_data dir is created(to segregate the manifest data), and all the tmp files are dumped there.